src/fiatconnect/fiatAccountSchemas/mobileMoney.ts

```diff
@@ -0,0 +1,73 @@
+import { FiatAccountSchema, FiatAccountType } from '@fiatconnect/fiatconnect-types'
+import { getObfuscatedAccountNumber } from 'src/fiatconnect'
+import { FiatAccountFormSchema } from 'src/fiatconnect/fiatAccountSchemas/types'
+import { FiatAccountSchemaCountryOverrides } from 'src/fiatconnect/types'
+import i18n from 'src/i18n'
+
+export const getMobileMoneySchema = (
+  implicitParams: {
+    country: string
+    fiatAccountType: FiatAccountType
+  },
+  countryOverrides?: FiatAccountSchemaCountryOverrides
+): FiatAccountFormSchema<FiatAccountSchema.MobileMoney> => {
+  const overrides = countryOverrides?.[implicitParams.country]?.[FiatAccountSchema.MobileMoney]
+  const mobileValidator = (input: string) => {
+    const regex = overrides?.mobile?.regex ? new RegExp(overrides.mobile.regex) : /^[0-9+]+$/
```

Comment:
this needs to start with a `+` right? You can also do the length with regex. E.g.,
`/^+\d{3-15}$`.
Also, do we want to support overrides for this field? Seems like there are UI specific rules like adding a `+` automatically, which could cause issues if we push a regex without a `+` for example.